lib/in/create-package-summary.js

```diff
@@ -30,6 +31,15 @@ function createPackageSummary(moduleName, currentState) {
         return false;
     }
 
+    // Ignore specified '--ignore' package globs
+    const ignore = currentState.get('ignore');
+    if (ignore) {
+        const ignoreMatch = Array.isArray(ignore) ? ignore.some(ignoredModule => minimatch(moduleName, ignoredModule)) : minimatch(moduleName, ignore);
```

Comment:
Where and how would you do that?
Looks like depcheck just returns unused dependencies, I'm looking for a simple way to check for a match between specified --ignore deps and createPackageSummary iteration's moduleName.
Not sure how you would use depcheck for this, unless you meant performing this check in another function.